CSA.md

```diff
@@ -1,215 +1,221 @@
+    2.2 <span class="header_3">Suspension.</span>  If <span class="keyterms_link">Customer</span> (a) has an outstanding, undisputed balance on its account for more than 30 days after the <span class="orderform_link">Payment Period</span>; (b) breaches Section 2.1 (Restrictions on Customer); or (c) uses the Product in violation of the Agreement or in a way that materially and negatively impacts the Product or others, then <span class="keyterms_link">Provider</span> may temporarily suspend <span class="keyterms_link">Customer's</span> access to the Product with or without notice. However, <span class="keyterms_link">Provider</span> will try to inform <span class="keyterms_link">Customer</span> before suspending <span class="keyterms_link">Customer's</span> account when practical. <span class="keyterms_link">Provider</span> will reinstate <span class="keyterms_link">Customer's</span> access to the Product only if <span class="keyterms_link">Customer</span> resolves the underlying issue.
 
 3. ## Professional Services  
 
-    <span class="coverpage_link">Provider</span> will perform the <span class="orderform_link">Professional Services</span> as detailed in an Order Form, if any, and <span class="coverpage_link">Customer</span> will reasonably cooperate with <span class="coverpage_link">Provider</span> to allow the performance of <span class="orderform_link">Professional Services</span>, including providing <span class="coverpage_link">Customer</span> Content as needed. <span class="coverpage_link">Provider</span> is not responsible for any inability to perform the <span class="orderform_link">Professional Services</span> if <span class="coverpage_link">Customer</span> does not cooperate as reasonably requested.
+    <span class="keyterms_link">Provider</span> will perform the <span class="orderform_link">Professional Services</span> as detailed in a Cover Page, if any, and <span class="keyterms_link">Customer</span> will reasonably cooperate with <span class="keyterms_link">Provider</span> to allow the performance of <span class="orderform_link">Professional Services</span>, including providing <span class="keyterms_link">Customer</span> Content as needed. <span class="keyterms_link">Provider</span> is not responsible for any inability to perform the <span class="orderform_link">Professional Services</span> if <span class="keyterms_link">Customer</span> does not cooperate as reasonably requested.
```

Comment:
One of @tblt 's changes looks like it was not merged in, the "customer" in customer content shouldn't be a key term class
```suggestion
    <span class="keyterms_link">Provider</span> will perform the <span class="orderform_link">Professional Services</span> as detailed in a Cover Page, if any, and <span class="keyterms_link">Customer</span> will reasonably cooperate with <span class="keyterms_link">Provider</span> to allow the performance of <span class="orderform_link">Professional Services</span>, including providing Customer Content as needed. <span class="keyterms_link">Provider</span> is not responsible for any inability to perform the <span class="orderform_link">Professional Services</span> if <span class="keyterms_link">Customer</span> does not cooperate as reasonably requested.
```